src/nvim/buffer_defs.h

```diff
@@ -814,6 +814,8 @@ struct tabpage_S {
   frame_T         *(tp_snapshot[SNAP_COUNT]);    /* window layout snapshots */
   dictitem_T tp_winvar;             /* variable for "t:" Dictionary */
   dict_T          *tp_vars;         /* internal variables, local to tab page */
+  char_u          *tp_localdir;     /* absolute path of local directory or
+                                       NULL */
```

Comment:
`//`-style comment. Also use `char *`.

Note that surrounding style is inherited from Vim. You *don’t* need to keep it.